### ISSUE TITLE
NSDate convenience initializer shows weird dependency on summer time

### ISSUE THREAD

**Issue by Hout** [malcommac/SwiftDate#186]
When testing  this code:
```
                    let rome = Region(timeZoneName: .EuropeRome)
                    let date = NSDate(year: 2011, month: 10, day: 9, region: rome)
                    let interval = date.timeIntervalSinceReferenceDate
                    expect(interval.toString()) == "10y 9m 5d 22h"
```

It fails when executed in summer time and succeeds in winter time. 
In summer time it matches `"10y 9m 5d 21h"`.

**Comment by Hout** [comment]
Time zone evaluation should take place agains the date evaluated, not the current date.

**Comment by Hout** [comment]
Fixed in 979f3ad203dcdc8cbd21183acdb4feeaee952e4c
